.github/workflows/lsp4jakarta.yaml

```diff
@@ -21,11 +21,11 @@ jobs:
           java-version: 11
       - name: Build LSP4Jakarta JDT LS Extension with Maven
         working-directory: ./jakarta.jdt
-        run: xvfb-run -a mvn -B install --file pom.xml
+		run xvfb-run -a mvn -B install --file pom.xml
```

Comment:
Is it valid to change the spacing and remove the ":" ?

Reply:
Fixed